schema/auth.json

```diff
@@ -0,0 +1,176 @@
+{
+  "$id": "https://serverlessworkflow.io/schemas/0.7/auth.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Serverless Workflow specification - auth schema",
+  "type": "object",
+  "auth": {
+    "oneOf": [
+      {
+        "type": "string",
+        "format": "uri",
+        "description": "URI to a resource containing auth definitions (json or yaml)"
+      },
+      {
+        "type": "array",
+        "description": "Workflow auth definitions",
+        "items": {
+          "type": "object",
+          "$ref": "#/definitions/authdef"
+        },
+        "additionalItems": false,
+        "minItems": 1
+      }
+    ]
+  },
+  "required": [
+    "auth"
+  ],
+  "definitions": {
+    "authdef": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Unique auth name",
+          "minLength": 1
+        },
+        "scheme": {
+          "type": "string",
+          "description": "Defines the auth type",
+          "enum": [
+            "basic",
+            "bearer",
+            "oidc"
+          ],
+          "default": "basic"
+        },
+        "info": {
```

Comment:
In the end, I'm really not a big fan of the term `info`. IMHO, and according to some of my searches, `properties` is the right term to use in this case, or at least it's the nomenclature used on .NET Framework. WDYT?

Reply:
updated